scripts/generate-types.js

```diff
@@ -54,9 +56,16 @@ export namespace Webhooks {
 
     public on (event: 'error', callback: (event: Error) => void): void
     public on (event: '*' | string[], callback: (event: Webhooks.WebhookEvent<any>) => void): void
+    public on (event: '*' | string[], callback: (event: Webhooks.WebhookEvent<any>) => Promise<void>): void
     ${signatures.join('\n')}
 
     public sign (data: any): string
+    public verify (eventPayload: WebhookPayloadWithRepository, signature: string): boolean
+    public verifyAndReceive (options: { id: string, name: string, payload: any, signature: string }): Promise<void>
+    public receive (options: { id: string, name: string, payload: any }): Promise<void>
+    public removeListener (event: string, callback: (event: Webhooks.WebhookEvent<any>) => void): void
+    public removeListener (event: string, callback: (event: Webhooks.WebhookEvent<any>) =>Promise<void>): void
```

Comment:
`event` can be a `string` or an event of strings. In line 66, I think the return type is incorrect/

```diff
- (event: Webhooks.WebhookEvent<any>) => void)
+ (event: Webhooks.WebhookEvent<any>) => Promise<void>)
```